src/Illuminate/Filesystem/Filesystem.php

```diff
@@ -63,14 +63,15 @@ public function get($path, $lock = false)
      * Get the contents of a file as decoded JSON.
      *
      * @param  string  $path
+     * @param  int  $flags
      * @param  bool  $lock
      * @return array
      *
      * @throws \Illuminate\Contracts\Filesystem\FileNotFoundException
      */
-    public function json($path, $lock = false)
```

Comment:
Isn't this a breaking change? The $lock parameter is now third instead of second?